.github/workflows/codeql.yaml

```diff
@@ -51,4 +52,4 @@ jobs:
       - name: Autobuild
         uses: github/codeql-action/autobuild@v1
       - name: Perform CodeQL Analysis
-        uses: github/codeql-action/analyze@v1
\ No newline at end of file
+        uses: github/codeql-action/analyze@v1
```

Comment:
There is no any change, right?

Reply:
Yes, just add a blank line for the end of the file